ui/app/src/state/epics/itemVersions.ts

```diff
@@ -44,7 +44,7 @@ export default [
       ofType(fetchItemVersions.type, versionsChangeItem.type),
       withLatestFrom(state$),
       switchMap(([{ payload }, state]) => {
-        const service = (state.versions.config)
+        const service = (state.versions.isConfig)
```

Comment:
@rart actually isConfig is needed for this verification (use getContentHistory or getConfigurationHistory). Also for validation to disable itemSelector